File: v2/csharp/src/main/csharp/io/swagger/Model/NewsletterSettings.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Newslettersettings {
    

    
    public List<NewsletterSetting> NewletterSettings { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Newslettersettings {\n");
      
      sb.Append("  NewletterSettings: ").Append(NewletterSettings).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/android/src/main/java/io/swagger/client/model/NewsletterSettings.java
package io.swagger.client.model;

import io.swagger.client.model.NewsletterSetting;
import java.util.*;

import com.wordnik.swagger.annotations.*;
import com.google.gson.annotations.SerializedName;


@ApiModel(description = "")
public class NewsletterSettings  {
  
  @SerializedName("newsletter_settings")
  private List<NewsletterSetting> newsletterSettings = new ArrayList<NewsletterSetting>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  public List<NewsletterSetting> getNewsletterSettings() {
    return newsletterSettings;
  }
  public void setNewsletterSettings(List<NewsletterSetting> newsletterSettings) {
    this.newsletterSettings = newsletterSettings;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class NewsletterSettings {\n");
    
    sb.append("  newsletterSettings: ").append(newsletterSettings).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
